src/Ilios/WebBundle/Controller/IndexController.php

```diff
@@ -4,16 +4,25 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
+use Ilios\CliBundle\Command\UpdateFrontendCommand;
+
```

Comment:
This is hardly an issue, but is there a reason for the extra line here?  Just wondering...